docker/quarto/README.md

```diff
@@ -0,0 +1,31 @@
+# Quarto runner
+
+In this directory we define a dockerized environment to run Quarto.
```

Comment:
What's the features we get on top of the official Quarto Docker image?

Reply:
fair point, let me complete this